src/citrine/resources/api_error.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, failure_message: Optional[str] = None, property: Optional[str
         self.failure_id = failure_id
 
 
-class ApiError(DictSerializable):
+class ApiError(DictSerializable, typ="api_error"):
```

Comment:
DictSerializable is being modified to make objects w/o typ abstract according to ABC.  There is no type field in the object that comes in.  Might be reasonable to just migrate this to a Serializable[] object.